mqtt/handler.go

```diff
@@ -20,17 +21,33 @@ import (
 
 var _ session.Handler = (*handler)(nil)
 
-const protocol = "mqtt"
+const (
+	protocol                           = "mqtt"
```

Comment:
Keep `protocol` as a separate constant, since this is a block for logging constants.